README.md

```diff
@@ -1,8 +1,8 @@
-# LMA Proxy charm
+# COS Proxy charm
 
-This Juju machine charm that provides a single integration point in the machine world with the Kubernetes-based [LMA bundle](https://charmhub.io/lma-light).
+This Juju machine charm that provides a single integration point in the machine world with the Kubernetes-based [COS bundle](https://charmhub.io/cos-light).
```

Comment:
It’s “cos-lite”

Reply:
And "COS HA" eventually. Do the individual charms (especially the ones which will never be clustered) need to specify?